_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: dynamodb
+parent: Sources
+grand_parent: Pipelines
+nav_order: 3
+---
+
+# dynamodb
+
+The `dynamodb` source enables change data capture (CDC) which streams [DynamoDB](https://aws.amazon.com/dynamodb/) table events such as `create`, `update`, or `delete` to Data Prepper events in near real-time.
+
+The source includes two ingestion options to stream DynamoDB events:
+
+1. _Full Initial Snapshot_ during [point-in-time recovery (PITR)](https://aws.amazon.com/dynamodb/pitr/) which backs your table automatically. This requires both the PITR Snapshots and DyanmoDB streaming options are enabled on your DynamoDB table.
```

Comment:
> This requires both the PITR Snapshots and DyanmoDB streaming

I'm not sure this is accurate. I don't think you need streaming to use the PITR.